ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/AccessList.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ *  * Copyright ConsenSys AG.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ *  * the License. You may obtain a copy of the License at
+ *  *
+ *  * http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ *  * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *  * specific language governing permissions and limitations under the License.
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.hyperledger.besu.ethereum.core;
+
+import static java.util.Collections.emptyList;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.tuweni.bytes.Bytes32;
+
+public class AccessList extends ArrayList<Map.Entry<Address, List<Bytes32>>> {
```

Comment:
Do we have some practices guidelines around using inheritance rather than composition/aggregation ?

Reply:
The reason I did it this way was because I was really looking to do a type alias but java doesn't have them. This was the closest analog.